WORKING_GROUPS.md

```diff
@@ -1,11 +1,11 @@
 # Node.js Working Groups
 
 Node.js Working Groups are autonomous projects created by the
-[Technical Steering Committee (TSC)](https://github.com/nodejs/node/blob/master/GOVERNANCE.md#technical-committee).
+[Core Technical Committee (CTC)](https://github.com/nodejs/node/blob/master/GOVERNANCE.md#core-technical-committee).
```

Comment:
You mean, it doesn't contemplate that WGs can be created at the TSC level too? How about if we renamed this section to "Node.js Core Working Groups"? Not sure if we need to mention here that there is a level above Core (TSC and TSC chartered WGs).